src/main/java/com/flightstats/hub/filter/DataDogRequestFilter.java

```diff
@@ -52,28 +54,44 @@ public void filter(ContainerRequestContext request, ContainerResponseContext res
     public static void finalStats() {
         try {
             DataDogState dataDogState = threadLocal.get();
-            if (null == dataDogState) {
-                return;
-            }
+            if (null == dataDogState) return;
+
             ContainerRequestContext request = dataDogState.getRequest();
-            String endpoint = getRequestTemplate(request);
-            String channel = channelName(request);
-            String method = request.getMethod();
             long time = System.currentTimeMillis() - dataDogState.getStart();
-            String callTag = "call:" + method + endpoint;
-            if (StringUtils.isEmpty(endpoint)) {
+
+            Map<String, String> tags = new HashMap<>();
+            tags.put("method", request.getMethod());
+            tags.put("endpoint", getRequestTemplate(request));
```

Comment:
I am 99% sure we can remove 'endpoint'.  It wasn't specific enough, so now we use 'call' instead.